### PR TITLE
Javascript: standardize inheritance: domain

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @orangejenny 

I sent the next part of Javascript inheritance standardize. This time I changed the Domain section.

Please let me know if you have any questions.

Regards,
Łukasz

### CODE REVIEW

File: corehq/apps/domain/static/domain/js/case_search.js [hunk 1]
```diff
@@ -13,22 +13,22 @@ hqDefine('domain/js/case_search', [
 ) {
     var module = {};
 
-    var Property = function (name) {
+    var property = function (name) {
```
Comment (orangejenny) [inline corehq/apps/domain/static/domain/js/case_search.js]: Nitpick, but I'd call this `propertyModel`, it's confusing to have a `property` function and then a `property` variable a few lines below.

**Review by orangejenny — COMMENTED** [review]
This looks good, just one picky comment.

**Comment by lwyszomi** [comment]
@orangejenny updated

**Review by orangejenny — APPROVED** [review]
Thanks, will merge once tests finish.

**Pull request merged**